pkg/block/params/block.go

```diff
@@ -46,4 +46,5 @@ type Azure struct {
 	StorageAccessKey string
 	TryTimeout       time.Duration
 	PreSignedExpiry  time.Duration
+	Url              *string
```

Comment:
What is this?

Reply:
This is a config parameter for testing purposes to provide a custom URL to override the default URL template - added comment